index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="logo">
+        <a href="">
+        <h3>Blessing Oluchi</h3>
+        </a>
+
+        <button class="Hamburger">
+            <img  src="images/Hamburger2.png" alt="Hamburger">
+        </button>
+
+    </header>
+
+    <section class="header-Bg">
+        <div class="header-Page">
+        <h1>I'm Blessing, <br> Glad to see <br> you!</h1>
+        <p>I'm a software developer! I can help you <br>
+            build a product , feature or website Look <br>
+            through some of my work and <br>
+            experience! If you like what you see and <br>
+            have a project you need coded, don't <br>
+            hestiate to contact me.</p>
+
+        <h6 class="connect">LET'S CONNECT</h6>
+
+        <div class="socialMedia">
+            <a href=""><img src="images/twitter2.png" alt="twitter"></a>
+            <a href=""><img src="images/linkedIn2.png" alt="linkedIn"></a>
+            <a href=""><img src="images/medium2.png" alt="medium"></a>
+            <a href=""><img src="images/github2.png" alt="github"></a>
+            <a href=""><img src="images/angelList2.png" alt="angelList"></a>
+        </div>
```

Comment:
- [ ] Good job using `<a>` tag for each element :+1: , but I think your social media links should be a list of icons, please use a `<ul>` tag to group them, and `<li>` per icon :100: because it is semantically correct.